compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: '3.8'
+services:
+  # db:
```

Comment:
since this is going to be part of different pull request let's remove these comments to have clean commits